src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/RequestRejectionReasons.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Http
+{
+    public enum RequestRejectionReasons
```

Comment:
Is there a convention to name enums in plural? Personally I'd prefer `RequestRejectionReason`.